charts/orchestrator/templates/knative-uninstall-hook.yaml

```diff
@@ -0,0 +1,163 @@
+{{- if .Values.serverlessOperator.enabled }}
```

Comment:
Is there an indication if the operator is installed by helm or not? and specifically by this chart?
This isn't a direct concern for this PR, but when I review this, it hits me we might uninstall an operator installed outside of the scope of the chart.

Reply:
There's a label added by `app.kubernetes.io/managed-by=Helm`. However I was able to remove a subscription managed by helm even after deleting this label. I suspect that helm doesn't care what I remove since it keeps track of its resources regardless of what I do with them.

From what I noticed in my trials, if I try to deploy the same chart with different name and existing manifests exist (existing subscription or namespaces), helm will complain that these objects have references to other chart deployments and refuse to deploy.